### PR TITLE
protobuf 21.11

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by cho-m** [comment]
Cancelled the re-run I started in case we want to finalize a decision on #112792 first.

**Comment by cho-m** [comment]
Needs a rebase now that other PR was merged.

**Comment by scpeters** [comment]
should we remove the `long build` label to see if it works without it since we haven't revision-bumped any other formulae yet? I know this usually requires changes to other formulae, but it might be worth checking how it does in a short build

**Comment by scpeters** [comment]
qt bottle related build failures on linux here too

~~~
brew install --only-dependencies qt-mysql
brew install qt-mysql
brew install --only-dependencies qt-percona-server
brew install qt-percona-server
~~~

https://github.com/Homebrew/homebrew-core/pull/117526 and https://github.com/Homebrew/homebrew-core/pull/117816 are also seeing qt linux failures

File: Formula/protobuf.rb [hunk 1]
```diff
@@ -2,11 +2,10 @@ class Protobuf < Formula
...
-  revision 1
 
   stable do
-    url "https://github.com/protocolbuffers/protobuf/releases/download/v21.9/protobuf-all-21.9.tar.gz"
-    sha256 "c00f05e19e89b04ea72e92a3c204eedda91f871cd29b0bbe5188550d783c73c7"
+    url "https://github.com/protocolbuffers/protobuf/releases/download/v21.11/protobuf-all-21.11.tar.gz"
+    sha256 "f045f136e61e367a9436571b6676b94e5e16631a06c864146688c3aaf7df794b"
 
     # Fix build with Python 3.11. Remove in the next release.
     patch do
```
Comment (chenrui333) [inline Formula/protobuf.rb:11]: https://github.com/protocolbuffers/protobuf/pull/10403

**Comment by chenrui333** [comment]
> #117526 and #117816 are also seeing qt linux failures

also in this one as well, #117594

**Pull request closed without merging**